wormchain/mainnet/genesis.json

```diff
@@ -327,7 +327,7 @@
         "guardian_set_expiration": 86400
       },
       "consensusGuardianSetIndex": {
-        "index": 0
+        "index": 1
```

Comment:
shouldn't this be `2`, which is the currently active guardian set index?

Reply:
I believe this is a bootstrapping issue.  We cannot switch to a newer guardian set until a quorum of guardians have registered their validator address.  Otherwise, the chain will deadlock because there won't be enough validator nodes online for consensus.

Once a quorum of guardians have registered their validator addresses the consensus guardian set will automatically switch.  See 
[RegisterAccountAsGuardian](https://github.com/wormhole-foundation/wormhole/blob/dev.v2/wormchain/x/wormhole/keeper/msg_server_register_account_as_guardian.go#L15).

Reply:
ok sounds good